content/projects/gsoc/index.adoc

```diff
@@ -26,54 +26,79 @@ in completing their summer projects.
 
 == GSoC 2019
 
-We plan to participate in GSoC 2019.
-Stay tuned for announcements.
-
-* link:/projects/gsoc/2019/[GSoC 2019 page]
-* link:/projects/gsoc/2019/project-ideas[GSoC 2019 Project ideas]
-* link:/projects/gsoc/proposing-project-ideas[HOWTO: Propose a project idea]
-* link:/projects/gsoc/2019/schedule[GSoC 2019 schedule].
-* link:/projects/gsoc/2019/application[GSoC 2019 application draft]
+In 2019 we plan to apply to Google Summer of Code.
+Currently we are looking for project ideas and mentors
+(link:/blog/2018/12/26/gsoc-2019-call-for-mentors/[announcement]).
+All Jenkins contributors are welcome to participate.
+We already have a significant number of project ideas,
+they are listed link:/projects/gsoc/2019/project-ideas[here].
+
+If you are a student, you are welcome to explore existing project ideas towards the GSoC application phase.
+There are ways to reach out to mentors,
+and many projects have lists of newbie-friendly issues you can start from.
+Students are also welcome to propose their own project ideas.
 
 == Links
 
 * link:/projects/gsoc/students[Information for students]
 * link:/projects/gsoc/mentors[Information for mentors]
 * link:/projects/gsoc/roles-and-responsibilities[Roles and Responsibilities]
 * link:/sigs/gsoc[Jenkins GSoC Special Interest Group]
+* link:/projects/gsoc/proposing-project-ideas[HOWTO: Propose a project idea]
+* link:/projects/gsoc/2019/project-ideas[GSoC 2019 project ideas]
+* link:/projects/gsoc/2019/schedule[GSoC 2019 schedule]
+* link:/projects/gsoc/2019/application[GSoC 2019 application draft]
+
+== Contacts
+
+We use the link:/sigs/gsoc[GSoC SIG] for communications about GSoC.
+Projects may also have their own mailing lists, chats and meetings.
+See details on project pages.
+
+=== Mailing lists
 
-=== Contacts
+Public Google Group: link:https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public[jenkinsci-gsoc-all-public].
```

Comment:
Perhaps we need to state the purpose of this mailing list. Students often post to the mentor or the org admin mailing. We should say it here that we expect students to post to this one wrt the application process, questions, etc.